cawg_identity/examples/cawg.rs

```diff
@@ -83,13 +70,6 @@ mod cawg {
             None,
         )?;
 
-        let cawg_raw_signer = raw_signature::async_signer_from_cert_chain_and_private_key(
```

Comment:
This appears to be an exact duplicate of the code preceding this block.

Reply:
Also: Please use a different cert for CAWG vs C2PA.